portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataReportAPI.testcase

```diff
@@ -1053,4 +1053,81 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: LRAC-12919 | Automation ID: LRAC-13495 | Test Summary: Check that you can access data from all existing segments"
+    @priority = 4
+    test RequestToAccessDataForAllSegments {
+        var segmentsTitleList = "managers,Annual Revenue > $500K,Dynamic Segment Test";
```

Comment:
Hey @manuferreira, I think validating those segments that are created by the Demo property can be a bit dangerous. Currently, we are not able to delete properties in `tearDown`, but when this functionality comes back, if some other previous test deletes the property and then your test will fail.

I think you could create another segment and verify that the segments created in your test is showing up in the API response

Reply:
Done :heavy_check_mark: